RadarCovid/Data/WS/Models.swift

```diff
@@ -31,9 +31,9 @@ open class Response<T> {
     }
 
     public convenience init(response: HTTPURLResponse, body: T?) {
-        let rawHeader = response.allHeaderFields
+        let rawHeader = response.allHeaderFields as! [String: String]
```

Comment:
It is an insecure conversion from `[AnyHasable: Any]` to `[String: String]` original code was fine.

Reply:
For the love of Jobs... ! is forbidden...
`var header = [String: String]()
        for (key, value) in rawHeader {
            if let key = key as? String, let value = value as? String {
                header[key] = value
            }
        }`

This is the way

Reply:
Was fixed in 02f6237e73b4930faf07056419f955999c1dda32

Reply:
> Was fixed in [02f6237](https://github.com/RadarCOVID/radar-covid-ios/commit/02f6237e73b4930faf07056419f955999c1dda32)

Bad fixed... you cannot use `as!` because is a forced conversion. You must use always conditional.

Reply:
02f6237 does not use `as!`, and I don't totally agree about using *always* optional casting...

Anyway, fixed (again) in b53c6b0a9de3f1d03391a9db80b63a095ffda3d7